src/notation/view/searchpopupmodel.cpp

```diff
@@ -32,14 +32,22 @@ void SearchPopupModel::load()
     });
 }
 
-void SearchPopupModel::search(const QString& text)
+bool SearchPopupModel::search(const QString& text)
 {
     mu::engraving::EngravingItem* element = notation()->elements()->search(text.toStdString());
     if (element) {
+        if (element->isPage()) {
+            const mu::engraving::Page* p = static_cast<const mu::engraving::Page*>(element);
+            element = p->firstMeasure();
+            if (!element && !p->systems().empty() && !p->systems().front()->measures().empty()) {
+                element = p->systems().front()->measures().front();
+            }
+        }
         notation()->interaction()->select({ element }, SelectType::SINGLE);
```

Comment:
Here we select the element but do not end note input mode, meaning that we jump to a measure and select it, but if we continue to write music, we will jump back to where we were. I don't know whether or not this is desirable and would like some feedback from a design person